[2025-03-11T16:10:15.777+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingest_bronze_layer_dag.transform_silver_5mins_incremental scheduled__2025-03-11T16:05:00+00:00 [queued]>
[2025-03-11T16:10:15.781+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingest_bronze_layer_dag.transform_silver_5mins_incremental scheduled__2025-03-11T16:05:00+00:00 [queued]>
[2025-03-11T16:10:15.781+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-03-11T16:10:15.788+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): transform_silver_5mins_incremental> on 2025-03-11 16:05:00+00:00
[2025-03-11T16:10:15.790+0000] {standard_task_runner.py:57} INFO - Started process 362 to run task
[2025-03-11T16:10:15.793+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ingest_bronze_layer_dag', 'transform_silver_5mins_incremental', 'scheduled__2025-03-11T16:05:00+00:00', '--job-id', '618', '--raw', '--subdir', 'DAGS_FOLDER/ingesting_pipeline.py', '--cfg-path', '/tmp/tmp4uefwzyg']
[2025-03-11T16:10:15.891+0000] {standard_task_runner.py:85} INFO - Job 618: Subtask transform_silver_5mins_incremental
[2025-03-11T16:10:15.910+0000] {task_command.py:416} INFO - Running <TaskInstance: ingest_bronze_layer_dag.transform_silver_5mins_incremental scheduled__2025-03-11T16:05:00+00:00 [running]> on host c8e526380ef3
[2025-03-11T16:10:15.941+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Andrew Tran' AIRFLOW_CTX_DAG_ID='ingest_bronze_layer_dag' AIRFLOW_CTX_TASK_ID='transform_silver_5mins_incremental' AIRFLOW_CTX_EXECUTION_DATE='2025-03-11T16:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-11T16:05:00+00:00'
[2025-03-11T16:10:15.942+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-11T16:10:15.942+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /opt/***/dbt/playstudio_project && export WAREHOUSE_HOST=None && export WAREHOUSE_PASSWORD=None && export WAREHOUSE_USER=None && dbt run --select tag:silver_5mins_incremental --target prod']
[2025-03-11T16:10:15.949+0000] {subprocess.py:86} INFO - Output:
[2025-03-11T16:10:17.477+0000] {subprocess.py:93} INFO - [0m16:10:17  Running with dbt=1.7.18
[2025-03-11T16:10:17.585+0000] {subprocess.py:93} INFO - [0m16:10:17  Registered adapter: postgres=1.7.18
[2025-03-11T16:10:17.620+0000] {subprocess.py:93} INFO - [0m16:10:17  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-11T16:10:17.621+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-03-11T16:10:17.621+0000] {subprocess.py:93} INFO - - models.playstudio_project.example
[2025-03-11T16:10:17.640+0000] {subprocess.py:93} INFO - [0m16:10:17  Found 1 model, 2 sources, 0 exposures, 0 metrics, 401 macros, 0 groups, 0 semantic models
[2025-03-11T16:10:17.641+0000] {subprocess.py:93} INFO - [0m16:10:17
[2025-03-11T16:10:37.752+0000] {subprocess.py:93} INFO - [0m16:10:37
[2025-03-11T16:10:37.758+0000] {subprocess.py:93} INFO - [0m16:10:37  Finished running  in 0 hours 0 minutes and 20.11 seconds (20.11s).
[2025-03-11T16:10:37.758+0000] {subprocess.py:93} INFO - [0m16:10:37  Encountered an error:
[2025-03-11T16:10:37.759+0000] {subprocess.py:93} INFO - Database Error
[2025-03-11T16:10:37.759+0000] {subprocess.py:93} INFO -   connection to server at "None" (125.235.4.59), port 5432 failed: timeout expired
[2025-03-11T16:10:37.759+0000] {subprocess.py:93} INFO - 
[2025-03-11T16:10:39.397+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2025-03-11T16:10:39.414+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-03-11T16:10:39.417+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=ingest_bronze_layer_dag, task_id=transform_silver_5mins_incremental, execution_date=20250311T160500, start_date=20250311T161015, end_date=20250311T161039
[2025-03-11T16:10:39.449+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 618 for task transform_silver_5mins_incremental (Bash command failed. The command returned a non-zero exit code 2.; 362)
[2025-03-11T16:10:39.463+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-03-11T16:10:39.481+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
